NVD OS vs Application Vulnerabilities/nvd_os_app_custom_field.py

```diff
@@ -4,10 +4,12 @@
 import json
 import gzip
 import io
+import os
 from collections import defaultdict
 import sys
 
-token_variable = 'API_KEY' # replace with your actual token
+
+token_variable = os.environ.get('API_KEY')
```

Comment:
The environment variable should be `KENNA_API_KEY` and it matches all the examples in the `blog_samples` repo.  `API_KEY` is too generic.